ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -140,6 +140,14 @@ PMP_SS_RMM:
   - low_pressure_status
   - high_pressure_status
 
+PMP_CPC_RMM:
```

Comment:
Don't need to differentiate that it is a circulation pump, meaning you should be able to use `PMP_SS_RMM`

Reply:
the fields in CPC are also required for this entity. Is PMP_SS_RMM_CPC a valid name to add? I have updated to this for now.